content/en/functions/strings.ToLower.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The already documented "lower" function is an alias for strings.ToLower. You can remove this page.

You are welcome to add a second signature to the existing function. Here's an example:
https://gohugo.io/functions/replacere

I keep this link handy to find aliases: https://raw.githubusercontent.com/gohugoio/hugo/master/docs/data/docs.json

Reply:
Page removed and second signature added to the existing function.